modules/terraform/jsonplan/plan.go

```diff
@@ -0,0 +1,83 @@
+package jsonplan
+
+import (
+	"encoding/json"
+
+	"github.com/gruntwork-io/terratest/modules/testing"
+	"github.com/stretchr/testify/require"
+)
+
+// The types in here are a slightly modified version of the types in:
+// https://github.com/hashicorp/terraform/tree/master/command/jsonplan
+// Modifications at:
+// - resource.go:17
+// - resource.go:52
```

Comment:
If you are referencing line numbers here, you should link to the commit sha instead of `master`. These line numbers look off now (L17 is a blank line for example).

Also, it would help if there is a description of what and why it changed as well.

Reply:
NOTE: Unfortunately, we won't be able to use this code as this would be considered a modification of the source file, which from my understanding, under MPL 2.0 (which [terraform is licensed as](https://github.com/hashicorp/terraform/blob/master/LICENSE)) means that this source file must also be licensed under MPL 2.0.*

With that said, there is now a public library from hashicorp with this content (https://github.com/hashicorp/terraform-json), so this is no longer necessary.

*IANAL